composer.json

```diff
@@ -8,7 +8,9 @@
     "googlechromelabs/third-party-capital": "dev-main"
   },
   "require-dev": {
+    "brain/monkey": "^2.6",
```

Comment:
first time seeing brain/monkey

Reply:
I haven't used it in a long time, but I first heard about it several years back, and I've always kept it in mind in case a WordPress project comes around where loading the database would be overkill. It's super useful for testing smaller WordPress projects/plugins/libraries, so maybe give it a closer look!